examples/trusted-sync/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rust:latest as build
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN apt-get update && apt-get install -y musl-tools
+
+COPY ./crates/ /kona/crates
+COPY ./bin/ /kona/bin
+COPY ./examples/ /kona/examples
+COPY ./Cargo.toml /kona
+COPY ./Cargo.lock /kona
+COPY ./rustfmt.toml /kona
+COPY ./README.md /kona
```

Comment:
(dumb question) does glob matching not work, as to copy the full contents of the folder in one copy directive?

Reply:
We can add a dockerignore - this was just so we don't copy unneeded build artifacts. Ended up taking way less time since things like `target` were not copied. I don't think it's bad practice to be deliberate about what you're placing into an image like this.